N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
I don't know more that what the README says. It read trig and graphs are part of the callback. In any case, it is out of our jurisdiction and out of this PR for sure.